js/modifyRecurringContributionPage.js

```diff
@@ -0,0 +1,50 @@
+CRM.$(function () {
+  const payment_scheme_schedule = CRM.vars.membershipextras.payment_scheme_schedule;
+  hideUnnecessaryPaymentPlanFields();
+  addFuturePaymentSchemeSchedule();
+
+  function hideUnnecessaryPaymentPlanFields() {
+    const frequency = CRM.vars.membershipextras.recur_contribution['frequency_unit'];
+    const payment_scheme_fields_to_hide = ['Cycle Day', 'Frequency', 'Installments', 'Next Contribution'];
```

Comment:
@omarabuhussein  I understand that we do not have a variable name convention in JavaScript/jQuery for our extension development.  In other field in the ME extension, the camel case has been used in this function. Perhaps, we should use a camel case to keep variable consistent with other files?

Reply:
fixed